config/domain.yaml

```diff
@@ -53,19 +53,23 @@ networks:
     gateway:
     type:
 
+  bmc:
+    defined: false
+    interface: bmc
+    hostname: "<%= alces.nodename %>.bmc.mgt.<%= domain %>"
+    short_hostname: "<%= alces.nodename %>.bmc"
+    #ip: "10.11.200.<%= alces.index %>"  # XXX uncomment this line when index is working
+    ip: 
+    netmask: 255.255.0.0
+    network: 10.11.0.0
+    gateway: 0.0.0.0
+    bmcpassword: 'Scitjaf4'
```

Comment:
Have done on my local, will get the change committed when I've got this reinstall sorted!